notebooks/EOPF_Tutorials/Sentinel-1_L2_OCN_product_format_prototype.ipynb

```diff
@@ -0,0 +1,570 @@
+{
```

Comment:
<p>Remove that list of dependencies, you're not using datatree, nor eopf and it's fine to just keep the cell with the code for the imports, doesn't need to specify them twice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EOPF-Sample-Service/eopf-sample-notebooks/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>